deploy/runtime/ui/Battery Dispatch Automated FOM.txt

```diff
@@ -30,10 +30,11 @@ Selection
 0
 Items
 6
-3
+4
 Perfect look ahead
 One day look behind
 Look ahead to custom weather file
+PV smoothing
```

Comment:
Index Labels property on variables was overwriting Object properties Items list so PV Smoothing wasn't showing up. I committed the fix to this branch